pkg/kubectl/cmd/config/config.go

```diff
@@ -46,7 +46,7 @@ func NewCmdConfig(f cmdutil.Factory, pathOptions *clientcmd.PathOptions, streams
 			The loading order follows these rules:
 
 			1. If the --` + pathOptions.ExplicitFileFlag + ` flag is set, then only that file is loaded.  The flag may only be set once and no merging takes place.
-			2. If $` + pathOptions.EnvVar + ` environment variable is set, then it is used a list of paths (normal path delimitting rules for your system).  These paths are merged.  When a value is modified, it is modified in the file that defines the stanza.  When a value is created, it is created in the first file that exists.  If no files in the chain exist, then it creates the last file in the list.
+			2. If $` + pathOptions.EnvVar + ` environment variable is set, then it is used as a list of paths (normal path delimitting rules for your system).  These paths are merged.  When a value is modified, it is modified in the file that defines the stanza.  When a value is created, it is created in the first file that exists.  If no files in the chain exist, then it creates the last file in the list.
```

Comment:
should probably correct the double spaces after periods as well here.

Reply:
I noticed that convention being used, but didn't want to make a stylistic judgement in what was otherwise a purely grammatical PR. If the preference is not to use double-spaces, I'm happy to remove that as part of this PR, or leave it for later.

Reply:
getting PR attention in k8s is hard.

i'd fix it as part of this PR in a separate commit and rename the title of the PR to reflect both commits. but this is optional - only if you feel like this is a viable change.

the block that has the double spaces is also user facing.

```
1. If the --` + pathOptions.ExplicitFileFlag + ` flag is set, then only that file is loaded.  The flag may only be set once and no merging takes place.
			2. If $` + pathOptions.EnvVar + ` environment variable is set, then it is used as a list of paths (normal path delimitting rules for your system).  These paths are merged.  When a value is modified, it is modified in the file that defines the stanza.  When a value is created, it is created in the first file that exists.  If no files in the chain exist, then it creates the last file in the list.
			3. Otherwise, ` + path.Join("${HOME}", pathOptions.GlobalFileSubpath) + ` is used and no merging takes place.`),
		Run: cmdutil.DefaultSubCommandRun(streams.ErrOut),
```

i googled and it seems that the discussions about single space vs two spaces after a period are either rants, anecdotal or both, but i'm sure our tech-writers at sig-docs would probably go for single spaces.